parts/chapters/2.fodt

```diff
@@ -78973,9 +78973,9 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
         </draw:frame><text:span text:style-name="T61"><text:line-break/></text:span>Figure <text:sequence text:ref-name="refFigure25" text:name="Figure" text:formula="ooow:Figure+1" style:num-format="1">2.26</text:sequence>: Example Normalize Oil-Water Relative Permeability Curves</text:p>
       </draw:text-box>
      </draw:frame>Consequently, water influx will be inhibited leading to unreliable results in terms of water production, and potentially gas production if aquifers are present in the model.</text:p>
-    <text:p text:style-name="P449">As mentioned previously, entering an original curve that is not normalized will automatically resolve these type of issues. If a normalized curve is entered, then ensure that simulator can distinguish between SWL and SWCR etc., based on the variable definitions provided in <text:sequence-ref text:reference-format="category-and-value" text:ref-name="ref0">Error: Reference source not found</text:sequence-ref>, <text:sequence-ref text:reference-format="category-and-value" text:ref-name="ref0">Error: Reference source not found</text:sequence-ref>, and <text:sequence-ref text:reference-format="category-and-value" text:ref-name="ref0">Error: Reference source not found</text:sequence-ref>. </text:p>
-    <text:p text:style-name="P449">Finally, see the SCALCERS keyword in the PROPS section, that defines the options used in the re-scaling process, including the options for two point and three point scaling. Whatever option is selected, always verify that the de-normalized curves that will be used in the model are correct and reasonable.</text:p>
-    <text:p text:style-name="P446"/>
+    <text:p text:style-name="P447">As mentioned previously, entering a correct original curve that is not normalized will automatically resolve these type of issues. If a normalized curve is entered, then ensure that simulator can distinguish between SWL and SWCR etc., based on the variable definitions provided in <text:reference-ref text:reference-format="text" text:ref-name="REF_TABLE_OIL_WATER_RELATI__INITIONS_8_2">Table 8.6</text:reference-ref>,<text:s/><text:reference-ref text:reference-format="text" text:ref-name="REF_TABLE_GAS_OIL_RELATIVE__INITIONS_8_2">Table 8.7</text:reference-ref><text:s/>and<text:s/><text:reference-ref text:reference-format="text" text:ref-name="REF_TABLE_GAS_WATER_RELATI__INITIONS_8_2">Table 8.8</text:reference-ref>. </text:p>
```

Comment:
On my system there are a lot of page breaks in this area. Do you see that, too?

Reply:
Yes, looks like I broke the paragraph formatting during the rebase. The new commit should fix this.